### PR TITLE
[Fragment] Replace bind with setOutput function.

### PR DESCRIPTION
**Pull request by Laimiux**
## What

Simplifying `FeatureView` by replacing `bind` function with a `setOutput` one.

### CODE REVIEW

File: formula-android-compose/src/main/java/com/instacart/formula/android/compose/ComposeViewFactory.kt [hunk 1]
```diff
@@ -1,45 +1,29 @@
...
 import com.instacart.formula.android.FeatureView
 import com.instacart.formula.android.ViewFactory
+import com.jakewharton.rxrelay3.BehaviorRelay
 
 
 abstract class ComposeViewFactory<RenderModel> : ViewFactory<RenderModel> {
 
     override fun create(inflater: LayoutInflater, container: ViewGroup?): FeatureView<RenderModel> {
         val view = ComposeView(inflater.context)
-        var firstRender = true
```
Comment (Laimiux) [inline formula-android-compose/src/main/java/com/instacart/formula/android/compose/ComposeViewFactory.kt:17]: Moved in to formula fragment

**Comment by carrotkite** [comment]
### JaCoCo Code Coverage 79.38% :white_check_mark:
| Class | Covered | Meta | Status |
|:---|:---:|:---:|:---:|
| `com/instacart/formula/android/FeatureView` | 100% | 0% | :white_check_mark: |
| `com/instacart/formula/android/FormulaFragment` | 72% | 0% | :white_check_mark: |
| `com/instacart/formula/android/ViewInstance` | 100% | 0% | :white_check_mark: |


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: formula-android/src/main/java/com/instacart/formula/android/FeatureView.kt [hunk 2]
```diff
@@ -13,18 +13,11 @@ import io.reactivex.rxjava3.core.Observable
...
+ * @param setOutput A function called to apply [RenderModel] to the view.
  * @param lifecycleCallbacks Optional lifecycle callbacks if you need to know the Fragment state.
  */
 class FeatureView<RenderModel>(
     val view: View,
-    val bind: (State<RenderModel>) -> Cancelable?,
+    val setOutput: (RenderModel) -> Unit,
     val lifecycleCallbacks: FragmentLifecycleCallback? = null,
-) {
-    class State<RenderModel>(
```
Comment (Laimiux) [inline formula-android/src/main/java/com/instacart/formula/android/FeatureView.kt:24]: We don't need this `State` class anymore

File: formula-android/src/main/java/com/instacart/formula/android/FormulaFragment.kt [hunk 3]
```diff
@@ -134,4 +125,34 @@ class FormulaFragment : Fragment(), BaseFormulaFragment<Any> {
...
+        val view = featureView ?: return
+
+        val fragmentId = getFormulaFragmentId()
+        val environment = FormulaFragmentDelegate.fragmentEnvironment()
+
+        try {
+            val start = SystemClock.uptimeMillis()
+            view.setOutput(output)
+            val end = SystemClock.uptimeMillis()
+
```
Comment (alexanderbezverhni) [inline formula-android/src/main/java/com/instacart/formula/android/FormulaFragment.kt:140]: nit: extra newline

**Review by alexanderbezverhni — APPROVED** [review]
Looks good

**Pull request merged**